src/Routing/RouteBuilder.php

```diff
@@ -596,17 +596,23 @@ public function redirect($route, $url, array $options = [])
      * to the `Controller\Admin\Api\` namespace.
      *
      * @param string $name The prefix name to use.
-     * @param callable $callback The callback to invoke that builds the prefixed routes.
+     * @param array|callable $params An array of routing defaults to add to each connected route.
+     *   If you have no parameters, this argument can be a callable.
+     * @param callable|null $callback The callback to invoke that builds the prefixed routes.
      * @return void
      */
-    public function prefix($name, callable $callback)
+    public function prefix($name, $params = [], callable $callback = null)
```

Comment:
Isn't this adding $params in the middle or the arguments, before $callback?
And also $callback nullable would not be strictly BC.
Unless this class is never extended on user land site, of course.